scripts/list-build-dependencies.sh

```diff
@@ -99,23 +99,26 @@ function list_packages() {
 	case "$1" in
 
 		apt)
+			# Package repo: https://pkgs.org/
 			# Apt separates GCC into the gcc and g++ pacakges, the latter which depends on the prior.
 			# Therefore, we provide g++ in-place of gcc.
 			VERSION_DELIM="-"
 			if [[ "${COMPILER}" == "gcc" ]]; then
 				COMPILER="g++"
 			fi
-			PACKAGES=(libtool build-essential autoconf-archive libsdl1.2-dev libsdl-net1.2-dev libopusfile-dev libspeexdsp-dev)
+			PACKAGES=(libubsan[0-9] libasan[0-9] libtool build-essential autoconf-archive libsdl1.2-dev libsdl-net1.2-dev libopusfile-dev libspeexdsp-dev)
 			;;
 
 		dnf)
+			# Package repo: https://pkgs.org/
 			VERSION_DELIM="-"
-			PACKAGES=(libtool autoconf-archive SDL SDL_net-devel opusfile-devel speexdsp-devel)
+			PACKAGES=(libubsan[\ 0-9] libasan[\ 0-9] libasan-static libtool autoconf-archive SDL SDL_net-devel opusfile-devel speexdsp-devel)
```

Comment:
CentOS/RHEL 7 has the names below; so I was trying to cover all the bases (similar to the apt/debian [0-9] notation).  I added the space [\ ] knowing that CentOS8 and Fedora just use `libasan`.  

```
libasan-4.9.2-6.el7.x86_64.rpm | The Address Sanitizer runtime library
libasan2-5.3.1-6.1.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 5
libasan3-6.3.1-3.1.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 6
libasan4-7.3.1-5.16.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 7
libasan4-7.3.1-5.15.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 7
libasan4-7.3.1-5.13.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 7
libasan4-7.3.1-5.10.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 7
libasan5-8.3.1-3.1.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 8
libasan5-8.3.1-3.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 8
libasan5-8.2.1-3.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 8
libasan5-8.2.1-3.bs1.el7.x86_64.rpm | The Address Sanitizer runtime library from GCC 8
```

I've replaced it simply with `libubsan` `libasan` and `libasan-static`, because I'd rather this script be _"simple and obvious"_, even if it means not covering all the bases - as opposed to full of clever tricks to figure CentOS X vs Fedora Y, and so on.  (perhaps if I ran these at home, maybe an elegant approach would be more obvious).